File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientInfo.java
package kz.greetgo.sandbox.controller.model;

import java.util.Date;
import java.util.List;

public class ClientInfo {
  public int id;
  public String name;
  public String surname;
  public String patronymic;
  public Gender gender;
  public Date birthDate;
  public int charm;
  public List<ClientAddress> addresses;
  public List<ClientPhone> phones;
}

File: sandbox.client/front/ts/model/ClientInfo.ts
import {Address} from "./Address";
import {Phone} from "./Phone";
import {Gender} from "./Gender";

export class ClientInfo {
  public id: number/*int*/;
  public name: string;
  public surname: string;
  public patronymic: string;
  public gender: Gender;
  public birthDate: number/*long*/;
  public charmId: number/*int*/;
  public factAddress: Address;
  public regAddress: Address;
  public phones: Phone[];
}